bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/testcases/MainActivityTest.kt

```diff
@@ -0,0 +1,117 @@
+package net.gini.android.bank.sdk.exampleapp.ui.testcases
+
+import android.Manifest
+import android.app.Activity
+import android.app.Instrumentation
+import android.content.Intent
+import android.net.Uri
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.Intents.intending
+import androidx.test.espresso.intent.matcher.IntentMatchers.hasAction
+import androidx.test.espresso.intent.matcher.IntentMatchers.hasType
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.filters.LargeTest
+import androidx.test.rule.GrantPermissionRule
+import net.gini.android.bank.sdk.exampleapp.ui.MainActivity
+import net.gini.android.bank.sdk.exampleapp.ui.screens.CaptureScreen
+import net.gini.android.bank.sdk.exampleapp.ui.screens.HelpScreen
+import net.gini.android.bank.sdk.exampleapp.ui.screens.MainScreen
+import net.gini.android.bank.sdk.exampleapp.ui.screens.OnboardingScreen
+import org.hamcrest.Matchers.allOf
+import org.junit.After
+import org.junit.Before
+import org.junit.FixMethodOrder
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.MethodSorters
+
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+@RunWith(AndroidJUnit4::class)
+@LargeTest
+class MainActivityTest {
```

Comment:
[Suggestion] Regarding organising the tests: what do you thing of organising them in the same way you did in QAlity? For example you could have a file called `OnboardingScreenTests` which implements the test cases in [PM-18](https://ginis.atlassian.net/browse/PM-18). You don't have to implement all test cases now, just implement test case 8 ("Onboarding screen appear for 1st launch") which is actually already implemented in `test2_clickPhotoPaymentButton()`.

Reply:
I like this idea - I maybe can try implementing it but I have to do some research first. 😄

Reply:
@a-szotyori i am happy to share that I've implemented your suggested idea and now we have a separate test class called `OnboardingScreenTests` where the test-cases in [PM-18](https://ginis.atlassian.net/browse/PM-18) are implemented. 🥹
<img width="291" alt="image" src="https://github.com/gini/gini-mobile-android/assets/144695421/29cf3fdc-9551-452b-9be6-4d65f6f809b7">